elements/bulbs-video/components/revealed.js

```diff
@@ -100,6 +100,10 @@ export default class Revealed extends React.Component {
       filteredTags.push(this.props.targetCampaignNumber);
     }
 
+    if (this.props.targetCampaignId) {
+      filteredTags.push('campaign-' + this.props.targetCampaignId);
```

Comment:
This can take advantage of template strings ``campaign-${this.props.targetCampaignId}``